### PR TITLE
Issue1374

### PR DESCRIPTION
**Pull request by skc222**

### CODE REVIEW

File: DEPENDENCIES.rst [hunk 1]
```diff
@@ -55,7 +55,7 @@ Package                Minimum Version
 ====================   ==================
 doxygen (for docs)     1.7.6.1
 tcmalloc (for speed)   any
-Cython                 0.13.1
+Cython                 0.25+
```
Comment (katyhuff) [inline DEPENDENCIES.rst:58]: good

File: DEPENDENCIES.rst [hunk 2]
```diff
@@ -120,7 +120,7 @@ and (optionally):
 #. python-numpy  or  python3-numpy
 #. python-nose   or  python3-nose
 #. python-jinja2 or  python3-jinja2
-#. cython        or  cython3       
+#. cython        or  cython3
```
Comment (katyhuff) [inline DEPENDENCIES.rst:123]: Good. Please add (see note below), thus:

```
#. cython        or  cython3   (see note below)
```

File: DEPENDENCIES.rst [hunk 3]
```diff
@@ -165,6 +165,11 @@ To determine which version of Python is already installed on your computer, run:
 
    python -V
 
+
+Cython Note
```
Comment (katyhuff) [inline DEPENDENCIES.rst:169]: Good!

File: DEPENDENCIES.rst [hunk 4]
```diff
@@ -165,6 +165,11 @@ To determine which version of Python is already installed on your computer, run:
...
+^^^^^^^^^^
+If you get any error related to cython version then this maybe because debian stable version of cython is .23.4 but cyclus installation requires .25+. To get latest cython version installed please visit the `Cython Documentation`_.
```
Comment (katyhuff) [inline DEPENDENCIES.rst:171]: This line is too long, and will need to be line-wrapped so that it is many shorter lines. Let's go with a little rephrasing as well, just for grammar:

```
If you get an error related to an old Cython version, then this may be 
because the Debian stable version of Cython is 0.23.4. However, 
Cyclus requires .25+. To install the latest cython version, please 
visit the `Cython Documentation`_.
```

File: new.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import json
```
Comment (katyhuff) [inline new.py:1]: Please remove new.py from this pull request.

File: CMakeLists.txt [hunk 6]
```diff
@@ -337,6 +337,24 @@ IF(NOT CYCLUS_DOC_ONLY)
                         OUTPUT_VARIABLE core_version
                         OUTPUT_STRIP_TRAILING_WHITESPACE)
     ENDIF(NOT "${CORE_VERSION}" STREQUAL "")
+
```
Comment (katyhuff) [inline CMakeLists.txt:340]: These changes to CMakeLists.txt do not belong in this pull request. The pull request should be from a clean develop branch. As is, you have changes that apply to both issue #1369 and issue #1374 in this pull request. 

These three commits do not belong in this PR:
- 3c59474
- 303400c
 - 6468d9b

This commit is the only commit that should be included in this PR: c98440b

**Review by katyhuff — CHANGES_REQUESTED** [review]
Good start! I have suggested a little rephrasing. Also, 3 of the 4 commits in this pull request do not belong. Please start from a clean develop branch (with none of your other changes) and commit include only the changes related to #5  .

File: new.py (hunk 5 above)
Comment (skc222) [reply new.py:1]: Oh yes..I forgot to remove it. I will do it now.
Thanks!
